routes/patient/patientsymptoms.js

```diff
@@ -0,0 +1,64 @@
+const router = require('express').Router();
+let Symptom = require('../../models/patientsymptom.model');
+
+router.route('/').get((req, res) => {
+    Symptom.findById(req.params.id)
+        .then(patientsymptoms => res.json(patientsymptoms))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
```

Comment:
this route should be (/:id) to be able to retrieve the id by "req.params.id"

Reply:
OR line 5 should be "Symptom.find()" so all the symptoms will be sent back

Reply:
I will make all the changes